products-service/serverless.ts

```diff
@@ -0,0 +1,163 @@
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'products-service',
+  },
+  frameworkVersion: '2',
+  plugins: [
+    'serverless-webpack',
+    'serverless-offline',
+    'serverless-aws-documentation'
+  ],
+  custom: {
+    documentation: {
+      api: {
+        info: {
+          version: '1',
+          title: 'product-service-API',
+          description: 'Product Service API'
+        }
+      },
+      models: [{
+        name: 'Product',
+        description: 'Product model',
+        contentType: 'application/json',
+        schema: {
+          type: 'object',
+          properties: {
+            id: {
+              type: 'number',
+              description: 'Product id',
+            },
+            title: {
+              type: 'string',
+              description: 'Product title',
+            },
+            description: {
+              type: 'string',
+              description: 'Product description',
+            },
+            price: {
+              type: 'number',
+              description: 'Product price',
+            },
+            image: {
+              type: 'string',
+              description: 'Product image',
+            }
+          }
+        }
+      },
+      {
+        name: 'ProductList',
+        description: 'List of products',
+        contentType: 'application/json',
+        schema: {
+          type: 'array',
+          items: {
+            $ref: '{{model: Product}}'
+          }
+        }
+      },
+      {
+        name: 'ServiceError',
+        description: 'Service error',
+        contentType: 'application/json',
+        schema: {
+          type: 'object',
+          properties: {
+            statusCode: {
+              type: 'number',
+              description: 'Status code of error'
+            },
+            error: {
+              type: 'string',
+              description: 'Error message'
+            }
+          }
+        }
+      }]
+    },
+    'serverless-offline': {
+      httpPort: 9000
+    }
+  },
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    stage: 'dev',
+    region: 'us-east-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    getProducts: {
+      handler: 'handler.getProducts',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products',
+            cors: true,
+            request: {
```

Comment:
it seems this request param configuration should be moved to getProductsById lambda config block

Reply:
You are right. Thanks. Will fix.